src/frontend/src/binder/expr/mod.rs

```diff
@@ -41,7 +41,19 @@ impl Binder {
             }
             Expr::Row(exprs) => self.bind_row(exprs),
             // input ref
-            Expr::Identifier(ident) => self.bind_column(&[ident]),
+            Expr::Identifier(ident) => {
+                if ["session_user"]
```

Comment:
Also `current_schema`?

Reply:
I would like to unsupport it until we really need to use it :) Postgres has so many features that nobody knows whether it will be used one day.

Reply:
I just want to add a second example so this `.iter().any()` reads less confusing. I was wondering why not just `"session_user" == ident.xxx` until I realized it can extend to support other "parentheses-less functions".